pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -105,6 +105,25 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 	}
 	logger.Infof("set user %q bucket max to %d", p.cephUserName, maxBuckets)
 
+	maxObjects := options.ObjectBucketClaim.Spec.AdditionalConfig["maxObjects"]
+	maxSize := options.ObjectBucketClaim.Spec.AdditionalConfig["maxSize"]
+	if maxObjects != "" || maxSize != "" {
+		if maxObjects != "" {
+			_, errCode, err = cephObject.SetQuotaUserObjectMax(p.objectContext, p.cephUserName, maxObjects)
+		}
+		if errCode == 0 && maxSize != "" {
+			_, errCode, err = cephObject.SetQuotaUserMaxSize(p.objectContext, p.cephUserName, maxSize)
+		}
+		if errCode == 0 {
+			_, errCode, err = cephObject.EnableUserQuota(p.objectContext, p.cephUserName)
+		}
+		if errCode > 0 {
+			p.deleteOBCResource(p.bucketName)
+			return nil, err
+		}
+
```

Comment:
nit: remove blank lines before closing brackets

Reply:
will address in next PR

Reply:
Done